src/components/products/product-list/ko/runtime/product-list.html

```diff
@@ -20,12 +20,12 @@
     <!-- ko ifnot: working -->
     <div class="table-body animation-fade-in">
         <!-- ko foreach: { data: products, as: 'product' } -->
-        <div class="table-row">
+        <div class="products-grid-row">
             <div class="col-5 text-truncate">
                 <a data-bind="attr: { href: $component.getProductUrl(product) }, text: product.displayName"></a>
             </div>
             <div class="col-7">
-                <span data-bind="markdown: { source: product.description, truncateAt: 250 }"></span>
+                <span class="cell-inherit-color" data-bind="markdown: { source: product.description, truncateAt: 250 }"></span>
```

Comment:
The markdown will surround the text with a `<p>` tag which has the color set globally from the styles; therefore, the typography color will be overwritten without this fix.